examples/native63/.prettierrc.js

```diff
@@ -1,6 +0,0 @@
-module.exports = {
```

Comment:
why not bring back prettier to keep the formatting the way it was?

Reply:
I'm going to change the settings to how the code was, not how the prettier config was, think they got out of sync.